frontEnd/src/views/Signin.vue

```diff
@@ -115,26 +120,29 @@ export default {
       passwordVisible: false,
       errormessage: '',
     }
-  }, 
-  methods: {
+  },
+
+  methods:  {
     async auth(email, password) {
-      await getUserByEmail({ email, password })
-      .then(response => {
-        //build cookie - SUPER WEIRD EDGE CASE THING
-        let user = response.data;
-        this.$cookies.set(user.id); //cookie includes the ID
-        if(user.isAdmin){
-          this.$router.push(`/admin/home/${user.id}`)
-        }else{
-          this.$router.push(`/applicant/${user.id}`)
-        }
+      await firebase.auth().signInWithEmailAndPassword(email, password)
+      .then((user) => {
+        this.id = user.user.uid
+        getUserByID(this.id)
+        .then(response => {
+          let currentUser = response.data;
+          console.log(currentUser)
```

Comment:
Should remove console logs in the frontend. It's okay for now but will need to be removed before production.